Makefile

```diff
@@ -30,16 +34,20 @@ IMAGE := ${DEIS_REGISTRY}/deis/${SHORT_NAME}:${VERSION}
 all:
 	@echo "Use a Makefile to control top-level building of the project."
 
+# Containerized dependency resolution / initial workspace setup
+bootstrap: check-docker
+	${DEV_ENV_CMD} glide up
+
 # This illustrates a two-stage Docker build. docker-compile runs inside of
 # the Docker environment. Other alternatives are cross-compiling, doing
 # the build as a `docker build`.
 build: check-docker
 	mkdir -p ${BINDIR}/bin
-	docker run --rm -v ${PWD}:/app -w /app golang:1.5.1 make docker-compile
+	${DEV_ENV_CMD} make docker-compile
 
 # For cases where build is run inside of a container.
 docker-compile:
```

Comment:
Oh... I realize what you're saying now.  This arises from some inherent ambiguity in the name `docker-build`.  To some that may mean a build facilitated _via_ docker, while to others, it may mean a build task meant for execution _within_ docker.  I tried to minimize this ambiguity in the router by calling one `build` and the other `binary-build`.  But even if that's better it's not by much.  All this being said, any controversy or disagreement over how these two targets are named in relation to one another and where they execute is actually not introduced by this PR.  Those target names are inherited from master.  This shouldn't be a blocker to this PR.